Source/ZXing.Net.Mobile.Android/ZXing.Net.Mobile.Android.csproj

```diff
@@ -12,7 +12,10 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <RootNamespace>ZXing.Mobile</RootNamespace>
     <AssemblyName>ZXingNetMobile</AssemblyName>
-    <TargetFrameworkVersion>v4.0.3</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
```

Comment:
This is an interesting topic.  We really should have TargetFrameworkVersion match the nuspec file's target framework version.  I hesitate to set this to 6.0 at this point just due to backwards compatibility reasons, can we bump it to 4.1 for now and also change the nuspec files to match?